tests/mod.rs

```diff
@@ -197,35 +197,40 @@ fn should_work_for_indexed() {
 }
 
 #[test]
-#[should_panic(expected = "Error decoding field Indexed.0")]
+#[should_panic(expected = "Not enough data to fill buffer")]
+#[cfg(not(feature = "chain-error"))]
```

Comment:
I think it's possible to use `cfg_attr` to adjust the expected panic message based on the feature, instead of only running these tests when the feature is disabled.

Reply:
indeed, fixed through https://github.com/paritytech/parity-scale-codec/pull/242/commits/e597d83b5c0a20a802f1fc7e8f9a656d2a313170